runelite-client/src/main/java/net/runelite/client/plugins/bosslogger/BossLoggerPanel.java

```diff
@@ -0,0 +1,437 @@
+		Set<String> categories = Tab.categories;
+		for (String categoryName : categories)
+		{
+			createTabCategory(categoryName, c);
+		}
+	}
+
+	// Creates all tabs for a specific category
+	private void createTabCategory(String categoryName, GridBagConstraints c)
+	{
+		MaterialTabGroup thisTabGroup = new MaterialTabGroup();
+		thisTabGroup.setLayout(new GridLayout(0, 4, 7, 7));
+		thisTabGroup.setBorder(new EmptyBorder(4, 0, 0, 0));
+
+		JLabel name = new JLabel(categoryName);
+		name.setBorder(new EmptyBorder(8, 0, 0, 0));
+		name.setForeground(Color.WHITE);
+		name.setVerticalAlignment(SwingConstants.CENTER);
+
+		ArrayList<Tab> categoryTabs = Tab.getByCategoryName(categoryName);
+		for (Tab tab : categoryTabs)
+		{
+			// Only create tabs for enabled recording options
+			if (bossLoggerPlugin.isBeingRecorded(tab.getName()))
+			{
+				// Create tab (with hover effects/text)
+				MaterialTab materialTab = new MaterialTab("", thisTabGroup, null);
+				materialTab.setName(tab.getName());
+				materialTab.setToolTipText(tab.getBossName());
+				materialTab.addMouseListener(new MouseAdapter()
+				{
+					@Override
+					public void mouseEntered(MouseEvent e)
+					{
+						materialTab.setBackground(BUTTON_HOVER_COLOR);
+					}
+
+					@Override
+					public void mouseExited(MouseEvent e)
+					{
+						materialTab.setBackground(BUTTON_COLOR);
+					}
+				});
+
+				// Attach Icon to the Tab
+				AsyncBufferedImage image = itemManager.getImage(tab.getItemID());
+				Runnable resize = () ->
+				{
+					materialTab.setIcon(new ImageIcon(image.getScaledInstance(35, 35, Image.SCALE_SMOOTH)));
+					materialTab.setOpaque(true);
+					materialTab.setBackground(BUTTON_COLOR);
+					materialTab.setHorizontalAlignment(SwingConstants.CENTER);
+					materialTab.setVerticalAlignment(SwingConstants.CENTER);
+					materialTab.setPreferredSize(new Dimension(35, 35));
+				};
+				image.onChanged(resize);
+				resize.run();
+
+				materialTab.setOnSelectEvent(() ->
+				{
+					this.showTabDisplay(tab);
+					materialTab.unselect();
+					materialTab.setBackground(BACKGROUND_COLOR);
+					return true;
+				});
+
+				thisTabGroup.addTab(materialTab);
+			}
+		}
+
+		if (thisTabGroup.getComponentCount() > 0)
+		{
+			tabGroup.add(name, c);
+			c.gridy++;
+			tabGroup.add(thisTabGroup, c);
+			c.gridy++;
+		}
+	}
+
+	// Landing page (Boss Selection Screen)
+	private void createTabPanel(Tab tab)
+	{
+		currentTab = tab;
+		this.removeAll();
+
+		createTabTitle(tab.getBossName());
+
+		bossLoggerPlugin.loadTabData(tab);
+
+		this.add(title, BorderLayout.NORTH);
+		this.add(wrapContainer(createLootPanel(tab)), BorderLayout.CENTER);
+	}
+
+	private JLabel createIconLabel(String iconName)
+	{
+		JLabel label = new JLabel();
+		BufferedImage icon = null;
+		synchronized (ImageIO.class)
+		{
+			try
+			{
+				icon = ImageIO.read(getClass().getResourceAsStream(iconName));
+			}
+			catch (IOException e)
+			{
+				log.warn("Error getting resource icon: " + iconName, e);
```

Comment:
This is only a warn, and the catch performs no other actions, is a full stack trace necessary or could it be limited to `e.getMessage();`?